src/pages/contact.js

```diff
@@ -18,8 +18,24 @@ const Contact = () => {
     setEmail(e.target.value);
   };
 
-  const onSubmit = (e) => {
-    e.preventDefault();
+  const onSubmit = (ev) => {
+    //  if we decide to use ajax and not thirdparty recapture.
+    ev.preventDefault();
+    const form = ev.target;
+    const data = new FormData(form);
+    const xhr = new XMLHttpRequest();
+    xhr.open(form.method, form.action);
```

Comment:
Would also suggest using `fetch` native JS API since it has a lot more than what `XHR` has, and the code would be smaller than this.

Reply:
Check the comment at the top, I didn't use this method... it I was using, I would have to dis- activate reCapture ... which could lead to a lot of spams, not to mentions ones I have been getting from you already hahah ... I just copied and pasted the script from that service we are using.. in case we do our own reCapture...

Reply:
Alright